src/react-components/ory/user-settings-screen.tsx

```diff
@@ -153,12 +164,13 @@ const body = ({
         "oidc",
         "lookupSecret",
```

Comment:
Shouldn't this be `lookup_secret`?

Reply:
Looks correct.

Reply:
Yes, it needs to be `lookup_secret`. I fixed it and it also resolved the UI diff.